packages/next-swc/crates/next-core/src/next_config.rs

```diff
@@ -943,12 +943,7 @@ impl NextConfig {
     #[turbo_tasks::function]
     pub async fn resolve_extension(self: Vc<Self>) -> Result<Vc<ResolveExtensions>> {
         let this = self.await?;
-        let Some(resolve_extensions) = this
-            .experimental
-            .turbo
-            .as_ref()
-            .and_then(|t| t.resolve_extensions.as_ref())
-        else {
+        let Some(resolve_extensions) = this.experimental.resolve_extensions.as_ref() else {
```

Comment:
Can we still look for `experimental.turbo.resolve_extensions` here, and emit a warning about deprecation while still accepting the old config option? @timneutkens: thoughts?

Reply:
Addressed [here](https://app.graphite.dev/github/pr/vercel/next.js/69108/Throw-warning-when-using-previous-resolveExtensions-config).